### PR TITLE
Start of edit place journey

### PR DESCRIPTION
**Pull request by joefitter**
This is a WIP

* added place page router with subrouters for confirm, delete, edit, success, update and view
* added views for confirm edit index and success
* added reusable form component
* added diff component

TODO:

* implement delete functionality
* implement create functionality
* add basic field level validation
* add error messages

### CODE REVIEW

**Comment by lennym** [comment]
There was a bug when editing an area with no NACWO defined. [I fixed it](https://github.com/UKHomeOffice/asl-service/pull/49/commits/cb557331fcee4b78fd29123463125398ed397445).

File: pages/places/index.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ module.exports = ({ content } = {}) => {
         comparator: 'AND',
         exact: true
       },
+      notes: {
+        show: true,
```
Comment (lennym) [inline pages/places/index.js]: I don't think this should be `show`n - it's appeared in the main body of the table.

Comment (lennym) [reply pages/places/index.js]: Also, does `expandable` have any meaning now? 👇

Comment (lennym) [reply pages/places/index.js]: Looks like "no", so I've removed this.

File: pages/place/routers/delete.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = settings => (req, res, next) => {
+  next();
+};
```
[the remark this answers is not in the document]
Comment (joefitter) [reply pages/place/routers/delete.js]: haha TODO

File: pages/place/fields/index.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  holding: {
+    label: 'Holding'
+  },
+  nacwo: {
+    label: 'NACWO'
+  }
+};
+
+export default (formatters = {}) => merge({}, fields, formatters);
```
[the remark this answers is not in the document]
Comment (joefitter) [reply pages/place/fields/index.jsx]: there _was_ - it used to contain formatters, now they are merged in - good spot

**Pull request merged**